db/migrate/20151101211016_add_reset_token_salt_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddResetTokenSaltToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :reset_token_salt, :string
```

Comment:
I don't think it's necessary to save the `reset_token_salt`. If you look at the `Encryptor` model I sent you, we can use Rail's `secret_key_base` to as the "salt" when hashing the token. Maybe this is something we can refactor once you get to that :smile: